Core/include/Acts/Surfaces/Surface.hpp

```diff
@@ -23,6 +23,7 @@
 #include "Acts/Utilities/Definitions.hpp"
 #include "Acts/Utilities/Intersection.hpp"
 #include "Acts/Utilities/Result.hpp"
+#include "Acts/Utilities/UnitVectors.hpp"
```

Comment:
Is this needed for the definition? If not, it should be moved to the `.cpp` or `.ipp` file.

Reply:
It's removed. The head file needed for the `transformBoundToFree` is moved to the `.ipp`.